src/components/Button/Button.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { ReactNode } from 'react'
+import styles from './Button.module.scss'
+
+interface ButtonProps {
+  onClick?: () => void
+  children: ReactNode
+}
+
+const Button: React.FC<ButtonProps> = ({ onClick, children }) => {
+  return (
+    <button className={styles.button} onClick={onClick}>
```

Comment:
```suggestion
interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement>  {
  children: ReactNode
}

const Button: React.FC<ButtonProps> = ({children, ...props }) => {
  return (
    <button className={styles.button} {...props}>
```

현재 새로 생성한 Button 컴포넌트에 기존 button element와 특별하게 다른 props를 사용하고 있지않아서 이렇게 해주면 더 공통적으로 사용하기 좋을 것 같습니다!
만약 onClick props가 기존 button element의 onClick과 타입이 다르다면 override를 해주면 됩니다!